src/cli/commands/public-cmds/commit-cmd.js

```diff
@@ -13,13 +13,20 @@ export default class Export extends Command {
     ['m', 'message <message>', 'commit message'],
     ['a', 'all', 'commit all new and modified components'],
     ['f', 'force', 'forcely commit even if specs fails and even when component was not changed'],
-    ['v', 'verbose', 'show specs output on commit']
+    ['v', 'verbose', 'show specs output on commit'],
+    ['i', 'ignore', 'ignore missing dependencies (default = false)']
```

Comment:
- [ ] remove the i shortcut, don't want to have a shortcut for this
- [ ] change the flag name to ignore-missing-dependencies